### DISCUSSION TITLE
默认订阅源

### DISCUSSION THREAD

**Discussion by Satrorn** [Guovin/iptv-api#765] [Q&A] [answered]
新版这个增加默认订阅源怎么设置？

**Comment by Guovin** [comment, marked as answer]
可修改config目录下的subscribe.txt文件

**Reply by Satrorn** [reply]
你说的默认源是自带的一些源吧,那我理解成别的的,我whitelist直接填源网址一直实现不了白名单功能,只能把频道接口一个个填到whitelist来实现,同时subscribe也要填相应的接口网址

**Reply by Guovin** [reply]
你是什么模式运行，把具体文件和日志发下

**Reply by Satrorn** [reply]
我是docker模式,但log文件都是一些测速的日志

**Reply by Satrorn** [reply]
是要发订阅源文件和白名单单文件给你复现吗？

**Reply by Satrorn** [reply]
按我理解,白名单上填的源网址,结果出来应的吗这个源网址内所有接口的后缀都是带白名单标记的,而不是订阅源

**Reply by Guovin** [reply]
我看了，白名单是生效的，只是对于白名单中某个订阅源下的多个接口并没有显示成白名单接口信息，显示了订阅源信息，不影响白名单功能正常运行，后续会修改

**Reply by Satrorn** [reply]
因为docker日志没什么直观的相关内容,结果又只显示订阅源,希望后续白名单,如果填频道接口的显示白名单,填订阅源网址的显示订阅源白名单,区分一下,也好排错

另外就是白名单内容,能否实现按填入内容先后排序,就是说如填了多个订阅源,都有东方卫视,那生成结果按白名单里订阅源的先后顺序来排序同一频道,不知我这样描述你能不能理解

**Reply by Guovin** [reply]
好的，了解

**Reply by Satrorn** [reply]
这两天测试了好多次,同一频道的排序,建议是: demo.txt(按原方式排第一顺位) –> whitelist.txt(按填入内容先后顺序排第二顺位) –> subscribe.txt(按测速结果排第三顺位),前面如果没內容就按后一个来,我觉得这样排序应该会好些

**Reply by Guovin** [reply]
模板中的接口会作为历史结果进行处理，除了含白名单标识的接口，都会参与测速。只有白名单接口才会优先排序前端。

**Reply by Satrorn** [reply]
嗯,那就白名单内的订阅源和单个频道接口标识区分下,排序按内容先后顺序排,先实现按自己像要的准确排序

**Reply by Guovin** [reply]
该问题已经处理，工作流已可以正常显示，gui和docker需等待下一次版本更新。

**Reply by Satrorn** [reply]
docker如要测试,我fork个推个版本也可以了吧

**Reply by Guovin** [reply]
因为这个只是显示优化，不影响白名单正常使用，暂时不构建更新镜像。若你想马上使用，可以自行构建。
